### PR TITLE
[KOGITO-5175] - [1.5.x] - Changes for productized build of Operator bundle

### PR DESCRIPTION
**Pull request by vaibhavjainwiz**
Many thanks for submitting your Pull Request :heart:! 

Below cmd needs to be executed to create prod operator bundle:

`make bundle-prod-build IMG=registry.redhat.io/rh-osbs/rhpam-7-rhpam-kogito-operator:7.11.0 BUILDER=osbs`

Please make sure your PR meets the following requirements:

- [ ] You have read the [contributors' guide](https://github.com/kiegroup/kogito-operator/blob/master/README.md#contributing-to-the-kogito-operator)
- [ ] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [ ] Pull Request contains a link to the JIRA issue
- [ ] Pull Request contains a description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster
- [ ] You've added a [RELEASE_NOTES.md](RELEASE_NOTES.md) entry regarding this change

### CODE REVIEW

**Review by ricardozanini — APPROVED** [review]

File: Jenkinsfile [hunk 1]
```diff
@@ -46,7 +46,7 @@ pipeline {
                 sh """
                     set +x && ${CONTAINER_ENGINE} login -u jenkins -p \$(oc whoami -t) --tls-verify=false ${OPENSHIFT_REGISTRY}
                     cd version/ && TAG_OPERATOR=\$(grep -m 1 'Version =' version.go) && TAG_OPERATOR=\$(echo \${TAG_OPERATOR#*=} | tr -d '"')
-                    ${CONTAINER_ENGINE} tag quay.io/kiegroup/rhpam-kogito-operator:\${TAG_OPERATOR} ${OPENSHIFT_REGISTRY}/openshift/rhpam-kogito-operator:pr-\$(echo \${GIT_COMMIT} | cut -c1-7)
+                    ${CONTAINER_ENGINE} tag registry.redhat.io/rh-osbs/rhpam-7-rhpam-kogito-operator:\${TAG_OPERATOR} ${OPENSHIFT_REGISTRY}/openshift/rhpam-kogito-operator:pr-\$(echo \${GIT_COMMIT} | cut -c1-7)
```
Comment (sutaakar) [inline Jenkinsfile:49]: The expected image tag format should be `registry.redhat.io/rhpam-7/rhpam-kogito-operator`

Comment (vaibhavjainwiz) [reply Jenkinsfile]: done

**Review by sutaakar — APPROVED** [review]

**Review by radtriste — APPROVED** [review]

**Comment by vaibhavjainwiz** [comment]
@radtriste With latest commit, bundle build is successful now. Could you please merge this PR?

**Review by almope — APPROVED** [review]
Thanks for the change!

**Pull request merged**